src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -59,6 +59,11 @@
         action = component.get("c.getAddresses");
         action.setParams({ hhId : hhId, listAddrExisting : null });
         action.setCallback(this, function(response) {
+
+		    // tell our visualforce page we are done loading
```

Comment:
@kbromer you must watch this video, and then tell us why you still want us to use spaces over tabs!!